ocs_ci/ocs/resources/drpc.py

```
@@ -28,19 +28,20 @@ def __init__(self, namespace, resource_name="", switch_ctx=None, *args, **kwargs
-                resource_name
-                if resource_name
-                else get_drpc_name(namespace, switch_ctx=switch_ctx)
-            ),
-            kind=constants.DRPC,
-            *args,
-            **kwargs,
-        )
+        with config.RunWithConfigContext(
+            switch_ctx if switch_ctx is not None else config.get_active_acm_index()
+        ):
+            super(DRPC, self).__init__(
```

Ruff (0.16.3)

[warning] Use `super()` instead of `super(__class__, self)`

Remove `super()` parameters

(UP008)